README.md

```diff
@@ -136,43 +141,47 @@ To run FEAT, give the full path of the fsf file as an input into FEAT_firstlevel
-    >
+    > 
     > If you choose to use manual alignment, run the script ./scripts/manual\_registration.sh secondlevel\_standard . Note that you will need to select "1" when prompted to make a new baseline, since this has not yet been created. However, once this is run you can reload the previously generated baseline and iteratively tweak alignment to improve the fit.
 
-    If you needed to make these changes, once you have a good alignment to standard, you can then map your results into standard space using the appropriate registration: flirt -in analysis/secondlevel_${Experiment}/${AnalysisName}/${Functional} -applyxfm -init analysis/secondlevel/registration.feat/reg/example_func2standard.mat -out analysis/secondlevel_${Experiment}/${AnalysisName}/${Output} -ref analysis/secondlevel/registration.feat/reg/standard.nii.gz.  
+   	If you needed to make these changes, once you have a good alignment to standard, you can then map your results into standard space using the appropriate registration: flirt -in analysis/secondlevel_${Experiment}/${AnalysisName}/${Functional} -applyxfm -init analysis/secondlevel/registration.feat/reg/example_func2standard.mat -out analysis/secondlevel_${Experiment}/${AnalysisName}/${Output} -ref analysis/secondlevel/registration.feat/reg/standard.nii.gz.  
+
+    If you want to use nonlinear methods like ANTs, and the scripts have been designed to use either alignment method (usually defaulting to check if nonlinear methods exist first). Specifically, we create a folder (`$SUBJ\_DIR/analysis/secondlevel/registration_ANTs`) that functions like `align_stats.sh` or `align_functionals.sh` uses to align data (and will use by default if it finds. To run these scripts, run the following from the `SUBJ_DIR`: `scripts/ants_registration/run_ANTs_highres2standard.sh $FREESURFER_ROOT`. `FREESURFER_ROOT` is the name of the FreeSurfer directory you want to use for the mask in this (masks from FreeSurfer are better than 3dSkullstrip). This should be the same scan as was used in step 10. This script first shows the mask that will be used in fslview. You should check that this mask is accurate. If there are parts of the skull are included in the mask or parts of the brain are excluded then ANTs may distort the brain in inappropriate ways. The script gives you the opportunity to stop the script and manually edit the mask (either on your local system or on the cluster). If you do this, you need to follow a few additional steps that the code describes and then rerun the script. Once the mask is set, the script will launch ANTs which takes 10-30 minutes. It first aligns to an age specific infant template and then linearly aligns to adult standard space. At the end it will show the outcome alignment (as well as manual linear alignment if that has also been performed). Note that you will need to set up ANTs to work on your end, including setting `ASHS_ROOT`   
```

Comment:
I would recommend a heavier rewrite to this section in which you upfront state that we do ANTs and then may be mention the alternatives in a note at the bottom

Reply:
Great idea -- done!